panel/ipywidget.py

```diff
@@ -0,0 +1,49 @@
+"""Functionality to enable easily interacting with ipywidgets via familiar APIs like pn.bind,
+@pn.depends and pn.rx"""
+
+from typing import TYPE_CHECKING, Iterable
+
+import param
+
+from .viewable import Viewer
+
+if TYPE_CHECKING:
+    try:
+        from traitlets import HasTraits
+    except ModuleNotFoundError:
+
+        class HasTraits:  # type: ignore
+            """Mock class"""
+
+else:
+
+    class HasTraits:  # type: ignore
+        """Mock class"""
+
+
+def to_viewer(widget: HasTraits, parameters: Iterable | None = None, **kwargs) -> Viewer:
+    """Returns a Viewer object with parameters synced to the ipywidget widget parameters
+
+    Args:
+        widget (HasTraits): The ipywidget to create the Viewer from.
+        parameters (Iterable | None): The parameters to add to the Viewer and to sync.
+            If no parameters are specified all parameters on the widget will be added
+            and synced.
+        kwargs: Any kwargs provided will be forwarded to the IPyWidget pane used to display the
+            widget.
+    """
+    raise NotImplementedError()
+
+
+def to_rx(
```

Comment:
See `pn.pane.ipywidgets._ipywidget_transform` for a traitlet -> param converter (currently only handling the `value`).

Reply:
Thx. I've used it.